test/spec/elem.txt

```diff
@@ -71,10 +71,10 @@ out/test/spec/elem.wast:527: assert_invalid passed:
   0000023: error: OnCallExpr callback failed
 out/test/spec/elem.wast:585: assert_trap passed: uninitialized table element
 out/test/spec/elem.wast:618: assert_invalid passed:
-  out/test/spec/elem/elem.61.wasm:000001f: error: type mismatch at elem segment. got externref, expected funcref
+  out/test/spec/elem/elem.61.wasm:000001f: error: type mismatch at elem segment. got funcref, expected externref
```

Comment:
https://github.com/WebAssembly/testsuite/blob/main/elem.wast#L618

The table contains externref, not funcref.